contracts/interfaces/IAccessLogs.sol

```diff
@@ -10,35 +10,16 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
-/// @title IMemoryAccessLog
+/// @title IAccessLogs
 
 pragma solidity >=0.8.0;
 
-interface IMemoryAccessLog {
-    enum AccessType {
-        Read,
-        Write
+interface IAccessLogs {
```

Comment:
Do you suggest moving it to another interface or library? If we move it to `IMetaStep` or `IUArchStep`, it may not make sense to import the interface in the `AccessLogs` library. If we move it to the `AccessLogs` library itself, any contract that want to use the structure will spend more deployment cost because all library functions are internal. What do you think?

Reply:
I think when all methods are internal, the compiler will strip all unused/unreachable/dead code. But I'm not sure! Do you know?

Reply:
You are right! I verified the deployment cost didn't increase at all. So I'll move the struct back to `AccessLogs` library.